src/cli_util.rs

```diff
@@ -580,15 +580,20 @@ impl WorkspaceCommandHelper {
     }
 
     pub fn base_ignores(&self) -> Arc<GitIgnoreFile> {
+        fn xdg_config_home() -> Result<String, VarError> {
+            if let Ok(x) = std::env::var("XDG_CONFIG_HOME") {
+                if !x.is_empty() {
+                    return Ok(x);
+                }
+            }
+            std::env::var("HOME").map(|x| format!("{x}/.config"))
```

Comment:
Actually, should this `format!()` be replaced by a `Path[Buf]::join()` to be correct on different platforms? Maybe it doesn't matter anymore because Windows is flexible about `/` and `\`?

Reply:
I don't think it matters much, but I'll swap in `Path` stuff for clarity. Had to move `expand_git_path` earlier but I think that's appropriate anyway.